### PR TITLE
Add new Footer design

### PR DESCRIPTION
**Pull request by denkravchu**
### Implemented:

- New Footer block design (https://www.figma.com/design/4704xY3b2arIRN2DpPBiED/FOOTER?node-id=0-1&t=RQ0bpgHhjMHgbBkl-1)

### Moments:
- Used website blue color tone instead of the one was used in design too keep the style clean
- Used website font family instead of the one was used in design to keep the style clean

Screenshot 2024-07-15 at 17 18 25

### CODE REVIEW

File: packages/frontend/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@testing-library/user-event": "^13.5.0",
     "d3": "^7.9.0",
     "express": "^4.18.2",
+    "framer-motion": "^11.3.2",
```
Comment (alexeyandreevsky) [inline packages/frontend/package.json:15]: Why add this module? Is it used somewhere?

File: packages/frontend/src/components/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                mr='auto'
+                h='auto'
+                alignItems='center'
+                justifyContent='space-between'
+            >
+                <p>pshenmic.dev</p>
+                {socialNetwork.length
+                  ? (<div className='Footer__WrapperNetwork'>
+                        {socialNetwork.map((item, i) => (
+                            <a
```
[the remark this answers is not in the document]
Comment (denkravchu) [reply packages/frontend/src/components/footer/index.js:27]: Link component from nextjs is used for the prefetching routes and routes navigation, while here we open external link. For external links you should use tag "a"

File: packages/frontend/src/components/footer/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                                key={i}
+                                className='Footer__Network'
+                                href={item.href ? item.href : '#'}
+                                target="_blank"
+                                rel="noopener noreferrer"
```
[the remark this answers is not in the document]
Comment (denkravchu) [reply packages/frontend/src/components/footer/index.js:32]: target="_blank" ensures the link opens in a new tab, and rel="noopener noreferrer" is a security best practice that prevents the new page from having access to the window.opener property, which can help protect against certain types of attacks

File: packages/frontend/src/components/footer/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (denkravchu) [reply packages/frontend/src/components/footer/index.js:27]: rel="noopener noreferrer" is a security best practice that prevents the new page from having access to the window.opener property, which can help protect against certain types of attacks

File: packages/frontend/src/components/footer/footer.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    @media screen and (max-width: $breakpoint-lg) {
+        flex-direction: column;
+        gap: 12px;
+     }
+
+    >:nth-child(1n){
+        flex: 1;
+    }
+
+    >p, a{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/footer/footer.scss:22]: Add spaces betwen > and {
`> p, a {`
In other places in the file too

File: packages/frontend/src/components/footer/footer.scss [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    align-items: center;
+    justify-content: space-between;
+    gap: 16px;
+
+    @media screen and (max-width: $breakpoint-lg) {
+        flex-direction: column;
+        gap: 12px;
+     }
+
+    >:nth-child(1n){
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/footer/footer.scss:18]: Is this the same as `> * {` ?

File: packages/frontend/src/components/footer/footer.scss [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+            }
+
+            >img{
+                position: relative;
+                width: 20px;
+                height: 20px;
+            }
+        }
+    }
+
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/footer/footer.scss:61]: Excess line

File: packages/frontend/src/components/footer/index.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  { img: '/images/icons/twitter.svg', href: 'https://x.com/Dashpay', alt: 'twitter', ariaLabel: 'Go to X(twitter)' },
+  { img: '/images/icons/discord.svg', href: 'https://discord.gg/GeH3ug5G', alt: 'discord', ariaLabel: 'Go to Discord' }
+]
+
+function Footer () {
+  return (
+        <Box px={3}>
+            <Flex
+                className={'Footer'}
+                maxW='1980px'
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/footer/index.js:16]: Wrap the component parameters in {}, please.
ml={'auto'} etc. in other places.

File: packages/frontend/src/components/footer/footer.scss [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+        flex: 1;
+    }
+
+    >p, a{
+        font-size: 12px;
+        color:  var(--chakra-colors-gray-250);
+        font-weight: 400;
+    }
+
+    .Footer__Copyright{
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/footer/footer.scss:28]: Space please

**Review by alexeyandreevsky — COMMENTED** [review]
Good job! Please add some code style changes.

File: packages/frontend/src/components/footer/footer.scss (hunk 5 above)
Comment (denkravchu) [reply packages/frontend/src/components/footer/footer.scss]: yes

File: packages/frontend/src/components/navbar/Navbar.js [hunk 9]
```diff
@@ -50,45 +50,48 @@ function Navbar () {
...
-            icon={isOpen ? <CloseIcon /> : <HamburgerIcon />}
-            aria-label={'Open Menu'}
-            display={{ lg: 'none' }}
-            onClick={isOpen ? onClose : onOpen}
-          />
+      <Flex
+        className={'Navbar'}
+        maxW='1980px'
+        ml='auto'
+        mr='auto'
```
Comment (alexeyandreevsky) [inline packages/frontend/src/components/navbar/Navbar.js:57]: Wrap parameters in braces please `{}`

**Review by alexeyandreevsky — APPROVED** [review]
Nice

**Review by pshenmic — APPROVED** [review]

**Pull request merged**